### DISCUSSION TITLE
Welcome to animate.css Discussions!

### DISCUSSION THREAD

**Discussion by eltonmesquita** [animate-css/animate.css#1429] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Emmauel98** [comment]
Please how do I use GitHub

**Reply by warengonzaga** [reply]
responded to your post.

**Reply by Ingrzs** [reply]
Fork and clone project.
Important reads readme.md
Contributing
Pull requests are the way to go here. We only have two rules for submitting a pull request: match the naming convention (camelCase, categorised [fades, bounces, etc]) and let us see a demo of submitted animations in a pen. That last one is important.

**Comment by iMukunth** [comment]
:root is not working in my code.can u please share the troubleshoot methods
